debian/pi-bluetooth.hciuart.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=Configure Bluetooth Modems connected by UART
 ConditionFileNotEmpty=/proc/device-tree/soc/gpio@7e200000/bt_pins/brcm,pins
-Requires=dev-serial1.device
-After=dev-serial1.device
+Requires=sys-devices-platform-soc-3f201000.serial-tty-ttyAMA0.device
+After=sys-devices-platform-soc-3f201000.serial-tty-ttyAMA0.device
```

Comment:
@punitagrawal Do you know a nicer way to do that ``sys-devices-platform-soc-3f201000.serial-tty-ttyAMA0.device`` ? It seems to complicate to me to be correct